TFNetworkRecLayer.py

```diff
@@ -6002,3 +6002,73 @@ def _get_output_native_rec_op(self, cell):
       y = y[0]
 
     return y
+
+
+class ZoneoutLSTMCell(BaseRNNCell):
+  """
+  Wrapper for tf LSTM to create Zoneout LSTM Cell.
+  This code is an adapted version of Rayhane Mamas version of Tacotron-2
+  """
+
+  def __init__(self, num_units, zoneout_factor_cell=0., zoneout_factor_output=0., name=None, **kwargs):
+    """
+    Initializer with possibility to set different zoneout values for cell/hidden states.
+    :param int num_units: 
+    :param float zoneout_factor_cell: 
+    :param float zoneout_factor_output: 
+    :param string name: 
+    :param dict kwargs: 
```

Comment:
`string` is not a valid type. Please check PEP (or PyCharm).
Also, the type of `name` is wrong. It looks like it can also be `None`. Then you should write `str|None`.
Also, `kwargs` will not be documented this way. Better just remove it.
And I think you need one empty line between the comment and the first param.